README.md

```diff
@@ -1 +1,32 @@
 # azuretpr
+
+Specification for Kubernetes clusters running in Giantnetes on Azure and managed by [azure-operator](https://github.com/giantswarm/azure-operator).
+
+## Getting Project
+
+Clone the git repository: https://github.com/giantswarm/azuretpr.git
+
+### How to build
+
+Build it using the standard `go build` command.
```

Comment:
I'm confused by myself now :D 

I think `$(glide novendor)` should be used for `go test`, but when building you should also build vendor, right?

If so the previous version was ok. Sorry for the confusion.